Pspcl/Pspcl.API/Program.cs

```diff
@@ -23,7 +22,12 @@ public static void Main(string[] args)
             if (app.Environment.IsDevelopment())
             {
                 app.UseSwagger();
-                app.UseSwaggerUI();
+                app.UseSwaggerUI(options=>
+                {
+                    options.SwaggerEndpoint("/swagger/v1/swagger.json", "v1");
+                    options.RoutePrefix = string.Empty;
+                    options.DocumentTitle = "My Swagger";
```

Comment:
Please change document title to something related to project. e.g. PSCPL Swagger, PSCPL APIs, etc.